docs/automated-tests/recording/overview.md

```diff
@@ -18,83 +20,83 @@ The below article will guide you trough the recoding process for web and WPF tes
+To start a recording session for a web test, enter the URL of the target page and the desired browser in the __Recording__ dialog.
 
 ![Enter URL, Choose browser][3]
 
 > **Tip**
 > <br>
 > <br>
-> If you plan to use a specific browser for the automation project, you can select that browser and enable the checkbox to __'Save my choice for the future'__ - this will <a href="/automated-tests/test-execution/quick-run-browsers#preferred-browser" target="_blank">set the browser as a preferred</a> for this project and you will no longer be prompted to select a browser for recording.
+> If you plan to use a specific browser for the test automation project, select the browser, and then select __Save my choice for the future__. This will <a href="/automated-tests/test-execution/quick-run-browsers#preferred-browser" target="_blank">set the browser as preferred</a> for the current project and the **Select Browser** prompt won't appear again.
 
-To launch the selected browser and navigate to the listed URL, click the __Record__ button in the _Recoding_ dialog. This will start a new instance of the browser and will launch the Test Studio extension, if applicable, Then the target page will be loaded and the <a href="/features/recorder/compact-recording-toolbar" target="_blank">**Compact Recording Toolbar**</a> will appear on top of the browser.
+To launch the selected browser and navigate to the desired URL, click __Record__ in the __Recording__ dialog. This will start a new instance of the browser and launch the Test Studio extension, if available. Next, the target page loads and the <a href="/features/recorder/compact-recording-toolbar" target="_blank">**Compact Recording Toolbar**</a> appears on top of the browser.
 
 ![Attached recorder][5]
 
-The Test Studio UI remains in the background of the browser, but once the page is loaded and the __Compact Recorder__ is attached, you can switch back to the project and check that a _Navigate_ step is recorded using the same URL as defined when starting the recording session.
+To verify that a **Navigate to** step was recorded, switch back to the project. The first step in the list points to the URL that you used to start the web test.
 
 ![Recorded navigate step][5a]
 
 > **Note**
 > <br>
 > <br>
-> Once a recording session is started, **do not start another instance of the same browser** until the session is finished!
+> Once a recording session starts, do not start another instance of the same browser until the session finishes!
 
-### WPF Test Recording
+### Record WPF Tests
 
-The WPF application is a desktop app and thus the automation process slightly differs when it comes to recording a test in Test Studio - you need to __define which is the application to test and where is its executable file__ - this is called <a href="/automated-tests/wpf/wpf-test" target="_blank">configuration of the WPF test</a>.
```

Comment:
You can combine lines 70-72 in a single sentence. These sentences repeat the same.